stable/prometheus-adapter/templates/custom-metrics-resource-reader-cluster-role.yaml

```diff
@@ -21,3 +21,13 @@ rules:
   - list
   - watch
 {{- end -}}
+{{- if .Values.rbac.psp }}
```

Comment:
If `rbac.enabled=false` this template will fail to render since it's outside the if statement. That said, I would prefer creating a separate `clusterrole-psp.yaml` and `clusterrolebinidng-psp.yaml` for this. Both should check if rbac.enabled as well.